src/pages/Home.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Tiles } from "@buildo/bento-design-system";
+import RestaurantPreview from "../components/RestaurantPreview";
+import rest_detail from "../mock-data/rest_detail.json";
+
+function Home() {
+  const name = rest_detail.name;
+  const imageUrl = rest_detail.image_url;
+  const rating = rest_detail.rating;
+  const address = rest_detail.location.address1;
+
+  const mockCards = [...Array(16).keys()].map((element) => {
+    return (
+      <RestaurantPreview
+        key={element}
+        name={name}
+        imageUrl={imageUrl}
+        rating={rating}
+        address={address}
+      />
+    );
+  });
+  return (
+    <Box
+      height="full"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      padding={16}
+    >
+      <Tiles space={16} columns={3} alignY={"bottom"}>
+        {mockCards}
+      </Tiles>
+    </Box>
```

Comment:
Do you actually need to specify `height` for this component? I get why you need it in the sidebar content, since it's reasonable to assume it would be smaller than the page content, but here the tiles should already be expanding as needed.

```suggestion
    <Inset space={16}>
      <Tiles space={16} columns={3} alignY={"bottom"}>
        {mockCards}
      </Tiles>
    </Inset>
```